app/assets/javascripts/dashboard/dataset_panel.js

```diff
@@ -0,0 +1,192 @@
+// @flow
+import * as React from "react";
+import { Row, Col, Card, Popover } from "antd";
+import Markdown from "react-remarkable";
+import TemplateHelpers from "libs/template_helpers";
+import _ from "lodash";
+
+import type { DatasetType } from "dashboard/dataset_view";
+
+const columnSpan = { xs: 24, sm: 24, md: 24, lg: 12, xl: 12, xxl: 8 };
+const thumbnailDimension = "500";
+const croppedDatasetCount = 6;
+
+type Props = {
+  datasets: Array<DatasetType>,
+  owningOrganization: string,
+  showOrganizationHeader: boolean,
+  limitDefaultDatasetCount: boolean,
```

Comment:
I think this should be a number and not a boolean, i.e. replace `croppedDatasetCount `